Microsoft.Toolkit.Uwp.Notifications/Microsoft.Toolkit.Uwp.Notifications.csproj

```diff
@@ -26,8 +26,21 @@
   <ItemGroup>
     <None Include="Microsoft.Toolkit.Uwp.Notifications.targets" Pack="true" PackagePath="build\native" />
   </ItemGroup>
-  
 
+  <ItemGroup Condition="!('$(TargetFramework)'=='net461')">
```

Comment:
Hmm it indeed doesn't work on .NET Core 3.0 WPF apps, I think because of this line which excludes the DesktopNotificationManager code on everything except .NET Framework apps

Reply:
The DesktopNotificationManager code also incorrectly gets included in ASP.NET Framework projects (when it should only be included in desktop WPF/console/client apps).

@michael-hawker is there any targeting we can do (on NuGet or in the compiled project) to only include the DesktopNotificationManager code for WPF (.NET Framework / .NET Core) and desktop apps, and NOT ASP.NET web projects?

Or do we simply need a separate NuGet package, Microsoft.Toolkit.Uwp.Notifications.Desktop? Separate package is okay with me, but it'd be neat if we could do it all in one and have it magically light up differently in all these different places